File: binder/src/models/description.rs
use sqlx::FromRow;

#[derive(Debug, FromRow)]
pub struct Description {
    pub description_id: i32,
    pub job_ad_id: i32,
    pub description_text: Option<String>,
    pub requirements: Option<String>,
    pub benefits: Option<String>,
    pub workstyle: Option<String>,
    pub about_project: Option<String>,
}

File: JobReadApi/JobReadApi.Application/Entities/Description.cs
namespace JobReadApi.Application.Entities;

public class Description
{
    public int DescriptionId { get; set; }
    public int JobAdId { get; set; }
    public string? DescriptionText { get; set; }
    public string? Requirements { get; set; }
    public string? Benefits { get; set; }
    public string? Workstyle { get; set; }
    public string? AboutProject { get; set; }

    public JobAd JobAd { get; set; } = null!;
}

